dbt/models/marts/users/dim_user_course_activity.sql

```diff
@@ -51,17 +51,17 @@ school_years as (
     from {{ ref('int_school_years') }}
 ),
 
-users as (
-    select 
-        user_id,
-        user_type
-    from {{ ref('dim_users') }}
-),
+-- users as (
+--     select 
+--         user_id,
+--         user_type
+--     from {{ ref('dim_users') }}
+-- ),
 
 combined as (
     select 
         ul.user_id,
-        u.user_type,
+        -- u.user_type,
```

Comment:
We need the user_type. This table should have both teacher and student activity, and we need to be able to differentiate and filter as their activity is very different. 
```suggestion
       u.user_type,
```